actionpack/lib/action_dispatch/railtie.rb

```diff
@@ -68,7 +68,7 @@ class Railtie < Rails::Railtie # :nodoc:
       config.action_dispatch.always_write_cookie = Rails.env.development? if config.action_dispatch.always_write_cookie.nil?
       ActionDispatch::Cookies::CookieJar.always_write_cookie = config.action_dispatch.always_write_cookie
 
-      ActionDispatch::Routing::Mapper.route_source_locations = Rails.env.development?
+      ActionDispatch::Routing::Mapper.route_source_locations = defined?(Rails::Command::RoutesCommand)
```

Comment:
Would it make sense to just move this line into `perform` of the Routes command?

Reply:
I thought about that, yeah. We could do that and keep the railtie a bit more clean-looking. 👍

Reply:
Oh, I remember why I avoided that now. `ActionDispatch::Routing::Mapper` isn't loaded before booting the application. That's probably why this was snuck into a railtie to begin with. I added a require to fix that.